waltz-ng/client/survey/survey-run-create.html

```diff
@@ -29,10 +29,10 @@
         <h4><strong>Issue Survey</strong></h4>
         <hr/>
 
-        Survey run '{{ctrl.surveyRun.name}}' has been saved.
+        Survey run <strong ng-bind="ctrl.surveyRun.name"></strong> has been saved.
         <br/>
         <br/>
-        Please click <a href="mailto:{{ctrl.surveyEmailRecipients}}?subject={{ctrl.surveyEmailSubject}}&body={{ctrl.surveyEmailBody}}">here</a> to generate survey invitation email.
+        Please click <a href="mailto:{{ctrl.surveyEmailHref}}">here</a> to generate survey invitation email.
```

Comment:
v. minor but why not generate the whole href (mailto included) and use `ng-href`  - will merge it anyway.